ParkingGarageRentals/Program.cs

```diff
@@ -67,7 +81,7 @@ static void Main(string[] args)
                         break;
                 }
 
-            } while (passNumberSelected != SentinelValue);
+            } while (isNotSentinelValue);
```

Comment:
Can you clarify what this outer loop does? The inner loop deals with the sentinel.

Reply:
I've now removed inner loop.